prowler/providers/aws/services/rds/rds_service.py

```diff
@@ -157,6 +160,49 @@ def __describe_db_snapshot_attributes__(self, regional_client):
                     f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
                 )
 
+    def __describe_db_clusters__(self, regional_client):
+        logger.info("RDS - Describe Clusters...")
+        try:
+            describe_db_clusters_paginator = regional_client.get_paginator(
+                "describe_db_clusters"
+            )
+            for page in describe_db_clusters_paginator.paginate():
+                for cluster in page["DBClusters"]:
+                    if not self.audit_resources or (
+                        is_resource_filtered(
+                            cluster["DBClusterIdentifier"], self.audit_resources
+                        )
+                    ):
+                        if cluster["Engine"] != "docdb":
+                            self.db_clusters.append(
+                                DBCluster(
+                                    id=cluster["DBClusterIdentifier"],
+                                    endpoint=cluster.get("Endpoint"),
+                                    engine=cluster["Engine"],
+                                    status=cluster["Status"],
+                                    public=cluster.get("PubliclyAccessible", False),
+                                    encrypted=cluster["StorageEncrypted"],
+                                    auto_minor_version_upgrade=cluster.get(
+                                        "AutoMinorVersionUpgrade", False
+                                    ),
+                                    backup_retention_period=cluster.get(
+                                        "BackupRetentionPeriod"
+                                    ),
+                                    cloudwatch_logs=cluster.get(
+                                        "EnabledCloudwatchLogsExports"
+                                    ),
+                                    deletion_protection=cluster["DeletionProtection"],
+                                    parameter_group=cluster["DBClusterParameterGroup"],
+                                    multi_az=cluster["MultiAZ"],
```

Comment:
Use always the `dict.get("value", default) please.

Reply:
The values that are not with get are always present in the API Call.